stable/kommander-karma/Chart.yaml

```diff
@@ -3,6 +3,7 @@ appVersion: "1.0"
 description: Kommander Karma
 name: kommander-karma
 home: https://github.com/mesosphere/charts
-version: 0.2.0
+version: 0.3.0
```

Comment:
is kommander-karma by any other chart _? Perhaps we should also update the chart that was using staging/karma

Reply:
yep, it'll be used by the kommander chart - but it depends on this pr getting merged so I was going to open up a new pr for kommander once this is out!